src/main/java/com/ldtteam/structurize/placement/handlers/placement/PlacementHandlers.java

```diff
@@ -256,7 +258,7 @@ public boolean canHandle(@NotNull final World world, @NotNull final BlockPos pos
           final boolean complete)
         {
             final List<ItemStack> itemList = new ArrayList<>(getItemsFromTileEntity(tileEntityData, world, pos));
-            itemList.add(BlockUtils.getItemStackFromBlockState(blockState));
+            itemList.add(BlockUtils.getItemStackFromBlockState(blockState, MapEnum.SURVIVAL_PLUS_BUILDING));
```

Comment:
I would like to have this enum at place where is structure placing initiated - like buildtool, builder etc.
Leaving this to Ray :)